docs/scaffold-a-new-project/README.md

```diff
@@ -0,0 +1,72 @@
+# Scaffold a New Project
+
+Neutrino provides a scaffolding tool to help you quickly jump-start a project. `create-neutrino-project` uses
+presets behind the scene to build projects. If you are not familiarized with them, take a moment to
+explore [what presets are](https://neutrino.js.org/presets/).
+
+## Getting Started
+Run the following command to start the interactive session. Substitute `<directory-name>` with the directory name you wish to create for this project.
+
+### Yarn
+
+```
+❯ yarn create neutrino-project <directory-name>
+```
+
+_Note: The `create` command is a shorthand that helps you do two things at once. See the [Yarn create docs](https://yarnpkg.com/lang/en/docs/cli/create/) for more details._
+
+### npm
+
+```
+❯ npm install -g create-neutrino-project
```

Comment:
You don't need to install `create-neutrino-project` globally to use it. When you have `npm` installed, you get 2 binaries: `npm` and `npx`. The `npx` one let's you use it without installing first:

```bash
npx create-neutrino-project <directory-name>
```

Reply:
@eliperelman I put `npm` because this is what the majority are familar with and have. Otherwise, we would need to tell them to install `npx` globally right? I'm just afraid making mention of `npx` might push developers with little JavaScript experience away.

```bash
npm install -g npx
npx create-neutrino-project <directory-name>
```

Reply:
No, having the latest npm also comes with npx, you get both. You don't have to install anything as long as you have the latest npm installed.

Reply:
Awesome, that's really nice!